packages/anac-certified-attributes-importer/src/index.ts

```diff
@@ -46,18 +48,23 @@ const loggerInstance = logger({
   correlationId,
 });
 
-await importAttributes(
-  sftpClient,
-  readModelQueries,
-  tenantProcess,
-  refreshableToken,
-  config.recordsProcessBatchSize,
-  config.anacTenantId,
-  loggerInstance,
-  correlationId
-);
+async function main(): Promise<void> {
+  try {
+    await importAttributes(
+      sftpClient,
+      readModelQueries,
+      tenantProcess,
+      refreshableToken,
+      config.recordsProcessBatchSize,
+      config.anacTenantId,
+      loggerInstance,
+      correlationId
+    );
+  } catch (error) {
+    loggerInstance.error(error);
+  } finally {
+    await cleanupResources(loggerInstance, drizzleCleanup);
+  }
+}
```

Comment:
```suggestion
try {
  await importAttributes(
    sftpClient,
    readModelQueries,
    tenantProcess,
    refreshableToken,
    config.recordsProcessBatchSize,
    config.anacTenantId,
    loggerInstance,
    correlationId
  );
} catch (error) {
  loggerInstance.error(error);
} finally {
  try {
    await cleanupResources(loggerInstance, drizzleCleanup);
  } catch (cleanupError) {
    loggerInstance.error(`Cleanup failed: ${cleanupError}`);
  }
}
```

Why we are not using try catch directly?

Furthermore, how are `cleanupResources` function errors handled?

I suggested to log with
```
try {
    await cleanupResources(loggerInstance, drizzleCleanup);
  } catch (cleanupError) {
    loggerInstance.error(`Cleanup failed: ${cleanupError}`);
  }
```
but maybe there's a better solution.

Reply:
I don't think logging is enough in this context; I just pointed out that the errors weren't being handled. I wouldn't even know exactly which errors should be handled — maybe for now it's better to leave it as you've done and then later evaluate how problematic this approach might be (if it turns out to be!).
I was just interested in thinking it through, but I see you've already considered it, so it's fine by me.